src/components/SwagProfile/SwagProfileComplete/styled.js

```diff
@@ -1,22 +0,0 @@
-import glamorous from 'glamorous';
-
-import v from '../../../styles/variables';
-
-export const SwagProfileComplete = glamorous.div({
-  backgroundColor: v.colorPrimaryDark,
```

Comment:
Nice job getting rid of these glamourous files!

Reply:
Thanks!